### PR TITLE
Add minutes for Jan 12 meeting

### PR DESCRIPTION
**Pull request by ivargrimstad**
Signed-off-by: Ivar Grimstad <ivar.grimstad@eclipse-foundation.org>

### CODE REVIEW

File: minutes/2021-01-12.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+- Ryan Cuprak (Jakarta EE Ambassadors)
+- Tanja Obradovic (Eclipse Foundation)
+- Werner Keil (Individual)
+
+## Agenda and Minutes
+
+### [IG] Individual Specifications Plan Reviews
+
+* Deadline for potential inclusion in 9.1: January 31
+*Ä None so far
```
Comment (nderwin) [inline minutes/2021-01-12.md:32]: An errant character here after the asterisk?

Comment (ivargrimstad) [reply minutes/2021-01-12.md:32]: Thanks :)

File: minutes/minutes.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Meeting Minutes
 
+## 2021
+
+* [2021-091-12](2021-01-12.md)
```
Comment (kwsutter) [inline minutes/minutes.md:5]: 2021-01-12 (not -091-).

**Review by kwsutter — COMMENTED** [review]
typo

**Pull request merged**